packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -653,11 +654,6 @@ export default class TradeStore extends BaseStore {
     @action.bound
     onUnmount() {
         this.disposeSwitchAccount();
-        this.proposal_info = {};
-        this.purchase_info = {};
-        WS.forgetAll('proposal');
```

Comment:
Let's keep the forget proposal call (and others like it) for `unMount` as we don't want the client to continue with its proposal request on route change.

Reply:
Yep, probably missed this initially as I was only thinking about removing code redundancy. Thanks.